home/migrations/0006_auto_20200623_2207.py

```diff
@@ -0,0 +1,254 @@
+# Generated by Django 3.0.5 on 2020-06-24 05:07
+
+from django.db import migrations, models
+import django.db.models.deletion
+import django.utils.timezone
+import uuid
+
+
+def migrate_appusers(apps, schema_editor):
```

Comment:
This function migrates AppUser -> Account/Device, and creates dummy placeholder records so that the AddField migrations can add foreign key references with valid data.